src/quo/components/navigation/page_nav/view.cljs

```diff
@@ -44,48 +44,55 @@
 
 (defn- right-section-spacing [] [rn/view {:style style/right-actions-spacing}])
 
+(defmulti add-button
+  (fn [{:keys [button-props]}]
+    (:content-type button-props)))
+
+(defmethod add-button :account-switcher
+  [{:keys [support-account-switcher? button-props]}]
+  (when support-account-switcher?
+    (let [{:keys [customization-color on-press emoji type]} button-props]
+      [rn/pressable {:on-press on-press}
+       [account-avatar/view
+        {:emoji               emoji
+         :size                32
+         :type                (or type :default)
+         :customization-color customization-color}]])))
+
+(defmethod add-button :default
+  [{:keys [background behind-overlay? button-props]}]
+  (let [{:keys [label icon-name]} button-props]
+    [button/button
+     (assoc button-props
+            :type       (button-type background)
+            :icon-only? icon-name
```

Comment:
We use the `icon-name` for `icon-name?` prop which expects a boolean value. 

I suggest wrapping : `(keyword? icon-name)` or `(boolean icon-name)`. Both are fine.